docs/templates/open-in-coder.md

```diff
@@ -13,105 +13,107 @@ Your browser does not support the video tag.
-  resource "coder_agent" "dev" {
-      # ...
-      dir = "~/${local.folder_name}"
-      startup_script =<<EOF
+locals {
+    folder_name = try(element(split("/", data.coder_parameter.git_repo.value), length(split("/", data.coder_parameter.git_repo.value)) - 1), "")
+}
 
-      # Clone repo from GitHub
-      if [ ! -d "${local.folder_name}" ]
-      then
-          git clone ${data.coder_parameter.git_repo.value}
-      fi
+resource "coder_agent" "dev" {
+    # ...
+    dir = "~/${local.folder_name}"
+    startup_script =<<EOF
 
-      EOF
-  }
-  ```
+    # Clone repo from GitHub
+    if [ ! -d "${local.folder_name}" ]
+    then
+        git clone ${data.coder_parameter.git_repo.value}
+    fi
 
-1. Embed the "Open in Coder" button with Markdown
+    EOF
+}
+```
 
-   ```md
-   [![Open in Coder](https://YOUR_ACCESS_URL/open-in-coder.svg)](https://YOUR_ACCESS_URL/templates/YOUR_TEMPLATE/workspace)
-   ```
+### 3. Embed the "Open in Coder" button with Markdown
 
-   > Be sure to replace `YOUR_ACCESS_URL` with your Coder access url (e.g.
-   > https://coder.example.com) and `YOUR_TEMPLATE` with the name of your
-   > template.
+```md
+[![Open in Coder](https://YOUR_ACCESS_URL/open-in-coder.svg)](https://YOUR_ACCESS_URL/templates/YOUR_TEMPLATE/workspace)
+```
 
-1. Optional: pre-fill parameter values in the "Create Workspace" page
+Be sure to replace `YOUR_ACCESS_URL` with your Coder access url (e.g.
+<https://coder.example.com>) and `YOUR_TEMPLATE` with the name of your template.
 
-   This can be used to pre-fill the git repo URL, disk size, image, etc.
+### 4. Optional: pre-fill parameter values in the "Create Workspace" page
 
-   ```md
-   [![Open in Coder](https://YOUR_ACCESS_URL/open-in-coder.svg)](https://YOUR_ACCESS_URL/templates/YOUR_TEMPLATE/workspace?param.git_repo=https://github.com/coder/slog&param.home_disk_size%20%28GB%29=20)
-   ```
+This can be used to pre-fill the git repo URL, disk size, image, etc.
 
-   ![Pre-filled parameters](../images/templates/pre-filled-parameters.png)
+```md
+[![Open in Coder](https://YOUR_ACCESS_URL/open-in-coder.svg)](https://YOUR_ACCESS_URL/templates/YOUR_TEMPLATE/workspace?param.git_repo=https://github.com/coder/slog&param.home_disk_size%20%28GB%29=20)
+```
 
-1. Optional: disable specific parameter fields by including their names as
-   specified in your template in the `disable_params` search params list
+![Pre-filled parameters](../images/templates/pre-filled-parameters.png)
 
-   ```md
-   [![Open in Coder](https://YOUR_ACCESS_URL/open-in-coder.svg)](https://YOUR_ACCESS_URL/templates/YOUR_TEMPLATE/workspace?disable_params=first_parameter,second_parameter)
-   ```
+### 5. Optional: disable specific parameter fields by including their names as
+
+specified in your template in the `disable_params` search params list
+
+```md
+[![Open in Coder](https://YOUR_ACCESS_URL/open-in-coder.svg)](https://YOUR_ACCESS_URL/templates/YOUR_TEMPLATE/workspace?disable_params=first_parameter,second_parameter)
+```
```

Comment:
Delete the section below, `## Devcontainer support`. 

Rationale: It's not (quite) related to Open in Coder and it's properly covered in `docs/templates/devcontainers.md`.